docs/troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+# Issues we met
+
+This document collects issues that we had with sssd-test-suite and their
+solution.
+
+## Ad machine sudden shutdown
+Probably license expired. Run
+
+    ./sssd-test-suite up -s ad ad-child && ./sssd-test-suite provision rearm
+
+
+## TASK [host : Install required vagrant plugins] failed due to vagrant dependencies
+
+remove ~/.vagrant.d
+uninstall vagrant-sshfs & vagrant-libvirt rpms and rerun host provisioning
+
+    rm -rf ~/.vagrant.d
+    sudo dnf remove vagrant-sshfs vagrant-libvirt
+    ./sssd-test-suite provision host -p "$POOL_DIR"
+
+
+## Various errors on fc28
+sssd-test-suite needs new ansibles, please upgrade to fc30.
+
+
+## error while downloading ad vagrant boxes during ./sssd-test-suite up -s
+
+An error occurred while downloading the remote file. The error
+message, if any, is reproduced below. Please fix this error and try
+again.
+
+SSL certificate problem: unable to get local issuer certificate
+
+Solution is to download ca chain for that website, store it
+into /etc/pki/ca-trust/source/anchors and run
+
+    update-ca-trust extract
+
+## ./sssd-test-suite rdp ad -- -g 90% ends immediatelly without any error message 
+
+Vagrant detects installed RDP client and it tries to use it.
+I might be that you don't have any RDP client installed or
+other brand than is expected by this documentation
+
+Solution is to use propper options for your RDP client.
+For example if you use xfreerdp, the comman should be
+
+    ./sssd-test-suite rdp ad -- /cert-ignore /size:90%
```

Comment:
I'd remove this part and add rdesktop to installed packages on host instead. I.e. here: https://github.com/SSSD/sssd-test-suite/blob/master/provision/roles/host/tasks/generic.yml#L1

Reply:
Well I have both installed and vagrant chose to use xfreerdp.

Unless we can force vagrant to pick rdesktop I would rather keep this hint

Reply:
Ok, please add information that this documentation expects rdesktop.